src/mym.cpp

```diff
@@ -459,14 +465,14 @@ void mexFunction(int nlhs, mxArray*plhs[], int nrhs, const mxArray*prhs[]) {
         //mysql_options(conn, MYSQL_OPT_RECONNECT, &my_true);
         if (!mysql_real_connect(conn, host, user, pass, NULL, port, NULL, CLIENT_MULTI_STATEMENTS))
             mexErrMsgTxt(mysql_error(conn));
-        const char*c = mysql_stat(conn);
-        if (c) {
+        const char*cs = mysql_stat(conn); // return connection status as a string - changed variable from c to cs

```

Comment:
This comment seems to be about the change rather than about the final status of the code.

Reply:
Just a comment to remind that mysql_stat(conn) returns connection status, done when tracing the code, got sick of single letter variable names :-)